tests/integration/security/testdata/beta-mtls-automtls.yaml

```diff
@@ -16,4 +16,16 @@ metadata:
     test-suite: "beta-mtls-automtls"
 spec:
   mtls:
-    mode: STRICT
\ No newline at end of file
+    mode: STRICT
+---
+apiVersion: "networking.istio.io/v1alpha3"
```

Comment:
this is not testing auto mtls right? if you are adding destinationRule?

Reply:
I made a mistake, i meant no adding rule here